account_invoice_report_due_list/tests/test_invoice_report_due_list.py

```diff
@@ -61,30 +62,27 @@ def setUpClass(cls):
         )
 
     def test_due_list(self):
-        invoice = self.env["account.invoice"].create(
+        invoice = self.env["account.move"].create(
```

Comment:
Better to use `Form` for avoiding not being resilient when other modules are installed.